helm/vertical-pod-autoscaler-app/values.yaml

```diff
@@ -1,189 +1,210 @@
-    pullPolicy: IfNotPresent
-    # recommender.image.tag -- Overrides the image tag whose default is the chart appVersion
-    tag: ""
-  # recommender.podAnnotations -- Annotations to add to the recommender pod
-  podAnnotations: {}
-  # recommender.podLabels -- Labels to add to the recommender pod
-  podLabels: {}
-  # recommender.podSecurityContetxt -- The security context for the recommender pod
-  podSecurityContext:
-    runAsNonRoot: true
-    runAsUser: 65534
-  # recommender.securityContext -- The security context for the containers inside the recommender pod
-  securityContext: {}
-  # recommender.resources -- The resources block for the recommender pod
-  resources:
-    limits:
-      cpu: 200m
-      memory: 3000Mi
-    requests:
-      cpu: 50m
-      memory: 500Mi
-  nodeSelector: {}
-  tolerations: []
-  affinity: {}
-
-updater:
-  # updater.enabled -- If true, the updater component will be deployed
-  enabled: true
-  # updater.extraArgs -- A key-value map of flags to pass to the updater
-  extraArgs:
-    # update.extraArgs.min-replicas -- Minimum number of replicas to perform update
-    min-replicas: 1
-    # update.extraArgs.v -- Verbosity level for updater
-    v: 2
-  replicaCount: 1
-  # updater.podDisruptionBudget -- This is the setting for the pod disruption budget
-  podDisruptionBudget: {}
-    # maxUnavailable: 1
-  image:
-    # updater.image.repository -- The location of the updater image
-    repository: giantswarm/vpa-updater
-    # updater.image.pullPolicy -- The pull policy for the updater image. Recommend not changing this
-    pullPolicy: IfNotPresent
-    # updater.image.tag -- Overrides the image tag whose default is the chart appVersion
-    tag: ""
-  # updater.podAnnotations -- Annotations to add to the updater pod
-  podAnnotations: {}
-  # updater.podLabels -- Labels to add to the updater pod
-  podLabels: {}
-  # updater.podSecurityContetxt -- The security context for the updater pod
-  podSecurityContext:
-    runAsNonRoot: true
-    runAsUser: 65534
-  # updater.securityContext -- The security context for the containers inside the updater pod
-  securityContext: {}
-  # updater.resources -- The resources block for the updater pod
-  resources:
-    limits:
-      cpu: 200m
-      memory: 3000Mi
-    requests:
-      cpu: 50m
-      memory: 500Mi
-  nodeSelector: {}
-  tolerations: []
-  affinity: {}
-
-admissionController:
-  # admissionController.enabled -- If true, will install the admission-controller component of vpa
-  enabled: true
-  # admissionController.extraArgs -- A key-value map of flags to pass to the admissionController
-  extraArgs:
-    webhook-timeout-seconds: 5
-  # admissionController.generateCertificate -- If true and admissionController is enabled, a pre-install hook will run to create the certificate for the webhook
-  generateCertificate: true
-  certGen:
+registry: quay.io
```

Comment:
you're not using this Variable? 
Please delete it if so.

Reply:
This variable is used only for crd-job in the crd-patch folder. But since I was doing some testing and deleted this file for this purpose it wasn't used anymore. 
I'll keep it since I'll revert my testing changes and that crd-job will be back.

Reply:
Well... this variable **has** to be supported, as it is set in China installs to use aliyun.
But its default value should be `docker.io`.
If upstream VPA chart does not support it, I guess you have to find a workaround :wink:

Reply:
All vpa components are using docker.io
The issue is that crd-job uses the `latest` tag for its `giantswarm/docker-kubectl` image which exists in quay.io but not in docker.io (I had a discussion with Jonas about this). 
So we could either use the quay.io registry for this one or fix the ci to have the `latest` tag in docker.io

Reply:
I've seen you've used docker.io with a tag, which looks like the right solution to me :)